.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,10 +1,19 @@
+## What this PR includes?
 <!-- 
-Please refer to the issue or JIRA ticket to close like `Closes #14` or 
-`Fixes [CLIENT-1]`
+Explain what this PR implements:
+In case you are fixing any specific issue, please refer to it with `Fixes #issue_number`.
+In case you are implementing a new feature, please write a brief description about it.
+Optionally, you can add any useful external reference that you think could help to documentate a bit more the implementation (for example: design/architecture documents, figma links, screenshots, etc.)
```

Comment:
```suggestion
Optionally, you can add any useful external reference that you think could help to documentate a bit more the implementation (for example: design/architecture documents, figma links, screenshots, etc.)
```
This sentence phrasing is a bit weird and `documentate` should be `document`.

Maybe it's easier to read if we rephrase it to:
```
As an optional step, you can link or add any useful external documentation to give more context about the proposed changes. (for example: design/architecture documents, figma links, screenshots, etc.)
```

Reply:
Done in https://github.com/decentraland/unity-renderer/pull/355/commits/b886306674708ad607224f5ac2f1906aa9b07c97, thanks!